src/renderer/controllers/subtitles-controller.js

```diff
@@ -137,3 +155,49 @@ function relabelSubtitles (subtitles) {
     track.label = counts[lang] > 1 ? (lang + ' ' + counts[lang]) : lang
   })
 }
+
+function detectVTTLanguage (buffer) {
+  var LanguageDetect = require('languagedetect')
+
+  // Detect what language the subtitles are in
+  var vttContents = buffer.toString().replace(/(.*-->.*)/g, '') // remove numbers?
```

Comment:
Re: your comment, I think this does remove numbers since it's removing everything on the line with the --> on it.